ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -1053,7 +1053,7 @@
     "ID": "caseProgressionReport",
     "Name": "Case progression report",
     "Description": "Cases at risk and Cases not meeting timetable",
-    "DisplayOrder": 30,
+    "DisplayOrder": 35,
```

Comment:
caseProgressionReport has the same display order as addCaseFlag is that correct?

Reply:
Yes just looked it is the same on the gatekeeping branch

Reply:
just rebased and the change is gone